src/hermezjs-typings.d.ts

```diff
@@ -90,14 +91,17 @@ declare module "@hermeznetwork/*" {
     // ToDo: This states are yet to be validated
     // state: "fged" | "fing" | "pend" | "invl";
     // ToDo: Are there more deposit types??
-    type: "Deposit" | "CreateAccountDeposit";
+    type: typeof TxType.Deposit | typeof TxType.CreateAccountDeposit;
```

Comment:
`typeof` is not needed here

Reply:
It's required because `TxType.Deposit` and `TxType.CreateAccountDeposit` are string literals, not types.  Without `typeof` TS infers the types as `any`:

![typeof](https://user-images.githubusercontent.com/15896177/136782904-c1d049c3-4ffa-44ed-97c2-d74f858345bb.gif)

Reply:
Now with enums, the `typeof` has been removed.